### ISSUE TITLE
MT documentation

### ISSUE THREAD

**Issue by rigaux** [project-asap/ASAP-documentation#1]
Hello,

My colleagues managed to install the WMT package. I can access the UI, but the lack of documentation at this point makes it impossible to go further.  Countless questions:
  - how can set the connection with IReS (I guess there should be one)
  - what are the datastores, nodes and links, how come that we just enter strings, and WMT magically understand something?
  - etc.

What would be nice is a step-by-step manual that would show how to create a basic workflow and execute it with IReS.  

Cheers

Philippe

**Comment by mhalkiad** [comment]
Yes, you are right Philippe, there is no documentation how to run a workflow in WMT. 
Until now, the integration between WMT and IReS is still in progress. 
But, you could use the following branches of WMT and IReS. 
Using this branches, you could see the list of the datasets and abstract operators and create a new abstract workflow in WMT and see it in IReS.

https://github.com/project-asap/workflow/tree/integration
https://github.com/project-asap/IReS-Platform/tree/project-asap-patch-3

**Comment by rigaux** [comment]
OK, thanks. But it order to make this UI more attractive, we would really need to explain what it does and how to obtain it. Now, it is quite opaque, and the design looks very cheap. 

We will focus on IReS, waiting for more user-friendly version to be integrated. Thanks again.
